modin/backends/pandas/query_compiler.py

```diff
@@ -32,6 +34,8 @@
     GroupbyReduceFunction,
 )
 
+import warnings
```

Comment:
I prefer this is grouped with external imports after line 23.

Reply:
fixed